SensioLabs/AnsiConverter/AnsiToHtmlConverter.php

```diff
@@ -41,7 +41,7 @@ public function convert($text)
     {
         // remove cursor movement sequences
         $text = preg_replace('#\e\[(K|s|u|2J|2K|\d+(A|B|C|D|E|F|G|J|K|S|T)|\d+;\d+(H|f))#', '', $text);
-        $text = htmlspecialchars($text, ENT_QUOTES, $this->charset);
+        $text = htmlspecialchars($text, ENT_QUOTES | ENT_SUBSTITUTE, $this->charset);
```

Comment:
Will change, but this will leave PHP 5.3 users with the somewhat bugged behavior of htmlspecialchars.